### PR TITLE
Git: Mark magna as kdh codeowner

### PR DESCRIPTION
**Pull request by PyroTechniac**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -13,4 +13,8 @@
 /src/commands/Anime/manga.ts @favna
 /src/commands/Misc/reddituser.ts @favna
 /src/commands/Pokemon/ @favna
-/src/lib/util/Pokedex.ts @favna
\ No newline at end of file
+/src/lib/util/Pokedex.ts @favna
+
+# Magna
+/src/routes/
```
Comment (favna) [inline .github/CODEOWNERS]:
```suggestion
/src/routes/ @gc 
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -13,4 +13,8 @@
...
+/src/lib/structures/api/
```
Comment (favna) [inline .github/CODEOWNERS]:
```suggestion
/src/lib/structures/api/ @gc 
```

**Review by favna — CHANGES_REQUESTED** [review]
These need `@gc` behind them. That's what actually adds the codeowns. The `#` is a comment. Syntax is similar to ignorefiles. 

https://github.com/gc

**Comment by cfanoulis** [comment]
yeah, that's all fun and games but no one cared to ask @gc

**Comment by cfanoulis** [comment]
and he (right??) agreed

uwu

**Review by kyranet — DISMISSED** [review]

**Review by kyranet — APPROVED** [review]

**Pull request merged**